inventory/components/InventoryTable.tsx

```
@@ -0,0 +1,281 @@
+                case 'Reorder alerts':
+                    filtered = filtered.filter(item => item.CurrentStock <= item.MinStock);
+                    break;
+                case 'Stock value':
+                    // Sort by stock value (CurrentStock * UnitPrice)
+                    filtered = filtered.sort((a, b) => {
+                        const valueA = a.CurrentStock * (a.UnitPrice || 0);
+                        const valueB = b.CurrentStock * (b.UnitPrice || 0);
+                        return valueB - valueA;
+                    });
+                    break;
+                case 'Stock quantity':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)